Documentation/Examples/RetriableSample.dib

```diff
@@ -0,0 +1,660 @@
+
+#!csharp
+
+var retryConfiguration = new RetryConfiguration();
+var retryOnRetryEnabledPolicy = new RetryOnRetryEnabledPolicy(retryConfiguration);
+
+AggregatePolicy aggregatePolicy = new AggregatePolicy()
+{
+    Policies = 
+    {
+        new RetryOnTooManyRequestsPolicy(),
+        retryOnRetryEnabledPolicy
+    }
+};
+
+// Change isRetryDisabled to true
+//retryConfiguration.isRetryDisabled = true;
+
+MockHttpService mockHttpService = new();
+Action retryAction = () => Retriable.Retry(() => mockHttpService.MakeLotsOfRequests(), CancellationToken.None, strategy: new Linear(TimeSpan.FromSeconds(2), 10), policy: aggregatePolicy);
+Task retryTask = Task.Factory.StartNew(retryAction);
+Thread.Sleep(5000); // Wait some time before disabling retry
+retryConfiguration.isRetryDisabled = true; // Disble retry via the config
+retryTask.Wait();
+
+#!markdown
+
+### `IRetryStrategy`
+
+The `IRetryStrategy` controls the way in which the operation is retried. It controls both the _delay between each retry_ and the _number of times that it will be retried_.
+
+There are several strategies provided out-of-the-box: `Count`, `DoNotRetry`, `Linear`, `Incremental`, and `Backoff`.
+
+#!markdown
+
+#### `Count`
+
+We've seen this retry strategy in the `IRetryPolicy` examples earlier.
+
+This simply retries a specified number of times, with no delay. `Retriable` operations default to a `new Count(10)` strategy.
+
+##### Example
+
+Reusing the `MockHttpService` from earlier, if we call the `MakeLotsOfRequests()` method (which always throws a "429 - Too Many Requests" `MockHttpServiceException` exception type) and supply a `RetryWhenTooManyRequestsPolicy` for the policy, and supply `new Count(10)` for the strategy, we should expect to see the method call retried 10 times. 
+
+#!csharp
+
+MockHttpService mockHttpService = new();
+Retriable.Retry(() => mockHttpService.MakeLotsOfRequests(), CancellationToken.None, strategy: new Count(10), policy: new RetryOnTooManyRequestsPolicy());
+
+#!markdown
+
+#### `DoNotRetry`
+This is the strategy equivalent of the `DoNotRetryPolicy`. It forces a retry to be abandoned, regardless of policy.
+
+##### Example
+
+Using the same example as in the `Count` retry policy above, the method will be called once and not retried.
+
+#!csharp
+
+MockHttpService mockHttpService = new();
+Retriable.Retry(() => mockHttpService.MakeLotsOfRequests(), CancellationToken.None, strategy: new DoNotRetry(), policy: new RetryOnTooManyRequestsPolicy());
+
+#!markdown
+
+#### `Linear`
+This retries a specified number of times (specified by `maxTries`), with a constant delay between each retry (specified by `periodicity`).
+
+For example, `new Linear(TimeSpan.FromSeconds(1), 5)` will retry up to 5 times. Each time it will  delay by 1s. So the first retry will be after 1s (wall clock time 1s), the second after another 1s (wall clock time 2s), the third after another 1s (wall clock time 3s).
```

Comment:
You've said the same thing it used to say in the docs until you picked it up 😉

It will *try* up to 5 times. The try will be immediate, the. The second try (if required) after 1s etc.